ir/state.cpp

```diff
@@ -13,9 +13,6 @@ namespace IR {
 
 State::State(const Function &f, bool source)
   : f(f), source(source), memory(*this) {
-  predecessor_data[&f.getFirstBB()].try_emplace(nullptr,
-                                                DomainTy(true, set<expr>()),
-                                                *this);
```

Comment:
The reason why I removed the predecesor_data for FirstBB was that it was storing an empty memory which made State::startBB(FirstBB) ovewrite the initial memory which contained initial values of const global vars. I made predecessor_data[FirstBB] store nothing and updated State::startBB.